ckanext/canada/tables/qpnotes.yaml

```diff
@@ -357,3 +357,123 @@ excel_column_heading_style:
   PatternFill:
     patternType: solid
     fgColor: FFEFEFEF
+
+- title: Proactive Publication - Question Period Notes Nothing to Report
```

Comment:
We need a corresponding entry in the .po for this

Reply:
I think I got that figured out now